Makefile

```diff
@@ -2,10 +2,15 @@ PROJECT := github.com/redhat-developer/odo
 GITCOMMIT := $(shell git rev-parse --short HEAD 2>/dev/null)
 PKGS := $(shell go list  ./... | grep -v $(PROJECT)/vendor)
 BUILD_FLAGS := -ldflags="-w -X $(PROJECT)/pkg/odo/cli/version.GITCOMMIT=$(GITCOMMIT)"
+DEBUG_BUILD_FLAGS := -ldflags="-X $(PROJECT)/pkg/odo/cli/version.GITCOMMIT=$(GITCOMMIT)"
```

Comment:
It would be better to separate common flags to the variable and than use it to define build flags and debug flags, so we don't have everything defined twice.

something like this
```
COMMON_LDFLAGS := -X $(PROJECT)/pkg/odo/cli/version.GITCOMMIT=$(GITCOMMIT)
BUILD_FLAGS := -ldflags=" -w $(LDFLAGS)"
DEBUG_FLAGS := -ldflags="$(LDFLAGS)"
```

Reply:
Done, thank you!

Reply:
Good stuff :wink: 

Us java folks are learning these new (or should I say old) tricks!